lib/Client.js

```diff
@@ -1019,6 +1042,15 @@ class Client extends EventEmitter {
         });
     }
 
+    /**
+    * Delete a guild scheduled event
+    * @arg {String} guildID The guild id of where the event belongs to
```

Comment:
```suggestion
    * @arg {String} guildID The guild ID of where the event belongs to
```

Please capitalise `ID` in other jsdoc locations as well